server.js

```diff
@@ -20,8 +20,9 @@ app.post("/login", (req, res) => {
         success: false
       });
     } else {
+      console.log("result isss: ", result);
```

Comment:
It's unclear what `result` should be here. You should try to ensure its type stays the same. Here it looks as though it can be `boolean` or an `object`.

Maybe you could try ensuring `result` is always of the form:

```js
{
  loggedIn: Boolean,
  name: String,
  userRole: String
}
```

or something like that - would be more readable imo.

Reply:
Good call! It's easy to forget about these throw away bits of code that add haha